content/user/apps/Flow-map.md

```diff
@@ -0,0 +1,46 @@
+<!--
+title: "Flow map"
+description: "Learn how to view the flow of data through your application."
+tags: "user UI applications flow map data access"
+-->
+
+The application flow map provides an interactive view of where data and resources are shared within your organization and beyond it.
+
+## How It Works
+
+Every time you exercise an application, the Contrast UI uses data reported from your Contrast agent to create a detailed map of how data flows from the application to other applications to which it's connected. The flow map organizes information for the application you're currently viewing into three connected sections: **Application Architecture**, **Back-End Systems** and **Connected Applications**. 
+
+By viewing all of the connections to your application, you can see the amount of data and resources that are associated with the application at full scale. By focusing on connections between individual systems and applications, you can also determine if users and connected applications in your organization have appropriate access to the current application and sensitive data associated with it. 
+
```

Comment:
The wording for this paragraph is still a work in progress. Any ideas are appreciated.

Reply:
> Every time you exercise an application, the Contrast UI uses data reported from your Contrast agent to create a detailed breakdown your application, the (components/systems?) within and the back-end systems it connects. As more applications are exercised and their back-end systems identified, shared ones are recognized and connected to help visualize how data flows between applications via these shared back-end systems. The flow map organizes information for the application you're currently viewing into three connected sections: **Application Architecture**, **Back-End Systems** and **Connected Applications**. 
> 
> By viewing all of the connections to your application, you can see the systems and resources that are associated with the application at full scale. By focusing on connections between individual systems and applications, you can better understand the nature of the network of services being built and utilized by a given application. So while this can more tactically show which libraries have vulnerabilities and verify documentation with regards to what an application is useing, it also starts to show a more comprehensive view of an organization.

Not sure if that's helpful, I kind of got going. It feels like more of a departure from what you originally had than I had initially planned. In general though, I knowt his is a _flow_ map, but I think we should avoid saying we're showing any kind of "data flowing" because we're not, we're just showing connections at this point. Who knows what data travels on them!?

Reply:
Just realized I was logged in on the wrong acct. :tableflip: